src/qt/window.cpp

```diff
@@ -312,8 +337,14 @@ wxWindowQt::~wxWindowQt()
 #if wxUSE_DRAG_AND_DROP
     SetDropTarget(NULL);
 #endif
+    if (m_qtWindow)
+    {
+        QtStoreWindowPointer( GetHandle(), NULL );
 
-    delete m_qtWindow;
+        // Delete QWidget when control return to event loop (safer)
+        m_qtWindow->deleteLater();
```

Comment:
I'm really not sure about this change. It seems wrong to have to do it, normally we shouldn't get any events after the window is destroyed. Doing this probably means that the sequence of events is not the same when using wxQt and in all the other ports.

Are you sure this is the right solution? Do we have a simple example showing what goes wrong without this commit?